testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1346,3 +1369,95 @@ def server_secret
   log "Reactivation Key: #{react_key}"
   step %(I enter "#{react_key}" as "reactivationKey")
 end
+
+When(/^I prepare Cobbler for the buildiso command$/) do
+  tmp_dir = "/var/cache/cobbler/buildiso"
+  $server.run("mkdir -p #{tmp_dir}")
+  # we need bootloaders for the buildiso command
+  _out, code = $server.run("cobbler mkloaders")
+  raise 'error in cobbler mkloaders, check the Cobbler logs' if code.nonzero?
```

Comment:
I'm curious, this command is not throwing in the "out" variable the logs we need to check?

Reply:
If we know at that point that we need to check Cobbler logs, just go ahead and check it, and print them :)

Reply:
Yes, if we have a raise here then we have to look at the Cobbler logs.

Reply:
Fixed.